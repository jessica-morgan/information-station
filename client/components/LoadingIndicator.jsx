import React from 'react'

const LoadingIndicator = () => {
  return (
    <div>
      <img src="/loading.gif"></img>
    </div>
  )
}

export default LoadingIndicator
