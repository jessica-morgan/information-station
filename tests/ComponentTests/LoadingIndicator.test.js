import React from 'reactn'
import LoadingIndicator from '../../client/components/LoadingIndicator'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

test('IsItBullshit renders correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <LoadingIndicator />
    </MemoryRouter>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('LoadingIndicator contains an image tag', () => {
  const wrapper = mount(<LoadingIndicator />)
  expect(wrapper.containsMatchingElement(<img src="/loading.gif"></img>)).toBeTruthy()
})
