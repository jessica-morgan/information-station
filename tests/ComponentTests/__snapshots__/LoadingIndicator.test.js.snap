// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IsItBullshit renders correctly 1`] = `
<div>
  <img
    src="/loading.gif"
  />
</div>
`;
